schema/virtual-memory-extension.asn

```diff
@@ -0,0 +1,19 @@
+Virtual-Memory-Extension
+DEFINITIONS
+   AUTOMATIC TAGS ::=
+BEGIN
+   EXPORTS VirtMem;
+
+   VirtMem ::= SEQUENCE {
+      -- 1 bit to indicate Svnapot standard extension is supported on Sv39, Sv48
+      -- and Sv57 page table entry.
+      svnapot BOOLEAN,
```

Comment:
```suggestion
      svnapot BOOLEAN DEFAULT FALSE,
```

This way people can omit this bit in the description if it's not implemented.

The same for the two BOOLEANs below.

Reply:
good point. done in the second commit